doc/living_in_belgium.md

```diff
@@ -0,0 +1,71 @@
+
+## Accommodation
+The first choice is where do you want to settle in? Most typical options are Brussels, Louvain la Neuve, Wavre, Namur, etc... Most of the people in the group live in Bruxelles. Talk to the lab members about this to have their opinions!
+If you are looking to rent your own place, try http://www.immoweb.be. You can filter by location, price, as well as "furnished" or "unfurnished". Note that if you take an unfurnished flat it's common to have absolutely NOTHING in it. No blinds, no light fixtures, potentially no kitchen! Note that kitchens are often described as “semi-equipped” or “hyper-equipped”, this refers to what type of appliances are there (dishwasher, fridge, etc).
+Some other websites to check if you are looking for a room/apartment where to live either in Brussels or in Louvain-la-Neuve:
+* https://www.facebook.com/groups/385369448291405/?fref=ts
+* https://www.facebook.com/groups/virtueljojo/
+* https://www.facebook.com/groups/195527370457521/
+* https://www.facebook.com/groups/188519461195986/
+* https://www.facebook.com/groups/134093889938198
+* https://www.facebook.com/groups/1798484627017398
+* http://www.appartager.be
+* http://www.expatarrivals.com/belgium/brussels/areas-and-suburbs-in-brussels
+
+
+### Temporary accommodation
+First of all, check the UCLouvain webpage for accommodation: https://uclouvain.be/en/study/logement-0.html You can see lots of options there. 
+
+[http://koterasmus.be/](https://koterasmus.be/) This is a group of students that are living together and share the project of welcoming, gathering and helping exchange students coming on the campus of Louvain-la-Neuve. Currently it doesn't seem very active though. 
+
+
+#### LOUVAIN-la-NEUVE
+If you want to benefit from a university accommodation, you have to make the request as soon as possible since they are always full. Ideally, for a start in September, the request should be done in March/April before starting. If you come from abroad the accommodation service of the university may ask for a contract that you don't have yet. In such a situation, ask Olivier to send a recommendation letter explaining you will be hired by providing the details and motivation, this may help unlock the situation.
+Website for UCLouvain accomodation : https://uclouvain.be/fr/etudier/logement/a-propos.html
+
+If you are coming for a short-term like 2-3 months, you should email them at least 1 or 1.5 months before arriving in Belgium to reserve a room. They answer very late sometimes, but if they don't answer for 1-2 weeks, send more emails highlighting that it's urgent.
+##### Alternatives: 
+
+- https://www.placet.be/index.php/en/for-residents/accommodation  
+- https://www.caleglln.com/ (This place belongs to Dominican friars and they are welcoming to international students who need short-term/long-term accommodation) 

```

Comment:
At the end of the accommodation section in the Google docs, there is a "Gas, Electricity, water" heading without anything under it and I didn't understand what is it. So I didn't include it here for now. If anyone knows any info about it, please add it.

Reply:
I put it back